content/docs/dvclive/api-reference/next_step.md

```diff
@@ -0,0 +1,52 @@
+# dvclive.next_step()
+
+Signals that the current step has ended. Check the
+[init parameters](/doc/dvclive/api-reference/init#parameters) for configuring
+the behavior.
+
+```py
+def next_step()
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+for step in range(3):
+    dvclive.log("metric", 0.9)
+    dvclive.next_step()
+```
+
+## Description
+
+Each call to `dvclive.next_step` will behave depending on the selected
+[init parameters](/doc/dvclive/api-reference/init#parameters) and whether `DVC`
+is available or not.
+
+If `summary` is True, on each `dvclive.next_step()` call, `dvclive` will
+generate a _summary_ of the metrics previously logged with `dvclive.log` and
+increase the `_step` count.
```

Comment:
What is `_step`? Looks special with `_`

Reply:
Nothing special, At some point I was thinking to use `_` to indicate that this was an internal property but given that the term `step` is already defined and used everywhere it probably doesn't makes sense